blog-service/2023-06-12-alerts.md

```diff
@@ -0,0 +1,14 @@
+---
+title: June 12, 2023 (Alerts)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - alerts
+  - monitors
+  - folders
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+New - We've added the ability to subscribe to alerts from monitor folders. When you subscribe at the folder level, you're automatically subscribed to all of its nested monitors and folders. This allows you to subscribe to multiple alerts at once, rather than subscribing to each individual monitor. [Learn more](/docs/alerts/monitors/alert-response).
```

Comment:
```suggestion
New - We've added the ability to subscribe to alerts from monitor folders. When you subscribe at the folder level, you're automatically subscribed to all of its nested monitors and folders. This allows you to subscribe to alerts from multiple monitors at once, rather than subscribing to each individual monitor. [Learn more](/docs/alerts/monitors/alert-response).
```